### PR TITLE
Core v0.6.2

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Screen Shot 2022-01-06 at 7 46 07 AM

**Comment by wanda-carlson** [comment]
This is wanda's message

**Comment by henry-richard** [comment]
This is Henry on Github

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # public
 
 This is a public repository managed by PaidPaper. The code is based in Python. It has several GitHub actions.
```
Comment (henry-richard) [inline README.md:3]: this line is weird

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
+edit 2
```
Comment (henry-richard) [inline README.md:4]: I'd change to edit 1

**Review by henry-richard — CHANGES_REQUESTED** [review]
Review comments please

**Review by henry-richard — APPROVED** [review]

**Pull request closed without merging**